.github/workflows/tests.yml

```diff
@@ -26,7 +27,8 @@ jobs:
               uses: shivammathur/setup-php@v2
               with:
                   php-version: ${{ matrix.php-version }}
-                  extensions: mbstring
+                  extensions: apcu, mbstring
```

Comment:
maybe we could also not install the extension to be sure there is no possible side effects on regular user installations

Reply:
https://docs.github.com/en/actions/learn-github-actions/expressions

Reply:
OK, I think that's done. I've also removed `ext-apcu` from `require-dev`.